base/logging.go

```diff
@@ -399,11 +399,13 @@ func Fatalf(logKey LogKey, format string, args ...interface{}) {
 
 // Errorf logs the given formatted string and args to the error log level and given log key.
 func Errorf(logKey LogKey, format string, args ...interface{}) {
+	StatsResourceUtilization().Add(StatKeyErrors, 1)
```

Comment:
Should we consider incrementing in Fatalf, Panicf?  Fatalf probably isn't too important (stats aren't going to be collected before process ends), but Panicf might be relevant for panics that are recovered.

I thought about moving this down to logTo, but I don't think that's worth the refactoring (to ensure the stats get incremented even if the log isn't enabled).